doc/classes/AStarGrid2D.xml

```diff
@@ -53,6 +53,15 @@
 				Clears the grid and sets the [member region] to [code]Rect2i(0, 0, 0, 0)[/code].
 			</description>
 		</method>
+		<method name="fill_region">
+			<return type="void" />
+			<param index="0" name="region" type="Rect2i" />
+			<param index="1" name="solid" type="bool" default="false" />
+			<param index="2" name="weight_scale" type="float" default="1.0" />
```

Comment:
This may be undesirable as `weight_scale` changes anyway. Even if the user just wants to enable/disable the region, rather than use this method to initialize the region. Since a negative `weight_scale` is invalid, we could use it as a "don't change" value and make `-1.0` the default value of the argument.

Reply:
The problem with this approach - if the user wants to specify just weight_scale - it will change the solid state anyway, so I've added two methods for solid and weight scale respectively.